MHM/technologies/culture_tech.txt

```diff
@@ -1183,7 +1183,7 @@ decolonization = {
 }
 wars_of_national_liberation = {
 	area = society_trends
-	year = 1955
+	year = 1950
```

Comment:
Change to 1960, I have a cleanup mechanic in V2ME Cleanup (MHM).txt that prevents crises from getting triggered before this tech gets researched by anyone.